lic/parameter/src/parameter_node.cpp

```diff
@@ -0,0 +1,45 @@
+#include <string>
+#include <ros/ros.h>
+#include <std_srvs/Empty.h>
+
+int main(int argc, char **argv)
+{
+	int red, green, blue;
+
+    // ROS节点初始化
+    ros::init(argc, argv, "parameter_config");
+
+    // 创建节点句柄
+    ros::NodeHandle node;
+
+    // 读取背景颜色参数
+	ros::param::get("/background_r", red);
+	ros::param::get("/background_g", green);
+	ros::param::get("/background_b", blue);
+
+	ROS_INFO("Get Backgroud Color[%d, %d, %d]", red, green, blue);
+
+	// 设置背景颜色参数
+	ros::param::set("/background_r", 255);
+	ros::param::set("/background_g", 255);
+	ros::param::set("/background_b", 255);
+
+	ROS_INFO("Set Backgroud Color[255, 255, 255]");
+
+    // 读取背景颜色参数
+	ros::param::get("/background_r", red);
+	ros::param::get("/background_g", green);
```

Comment:
**[cpplint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br> Tab found; better to use spaces  [whitespace/tab] [1]